src/locales/fr/translation.json

```diff
@@ -160,6 +160,7 @@
   "pager.of": "sur",
   "pager.rows": "Items par page:",
   "personalization": "Personnalisation",
+  "personalization.autoDetectColorScheme": "Synchroniser avec l'ordinateur",
```

Comment:
```suggestion
```
Doesn't look like this is used anymore after moving to a dropdown instead of an additional toggle/switch